app/app.rb

```diff
@@ -20,16 +20,14 @@ def run
       Hammeroids::Lobby.new.clear
 
       EventMachine.run do
-        EventMachine::WebSocket.start(host: @socket_host, port: @socket_port) do |ws|
 
+        channel = EM::Channel.new
+        @count = 0;
```

Comment:
If i understand correctly, i think count might be replaced by player UUID, but we could do that later.